cc/repositories.bzl

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2022 Swift Navigation Inc.
+# Contact: Swift Navigation <dev@swift-nav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+LLVM_DISTRIBUTION_URL = "https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.0/clang%2Bllvm-14.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz"
+
+def swift_cc_toolchain():
+    if "llvm-distribution" not in native.existing_rules():
```

Comment:
I've seen other things like this use [maybe](https://bazel.build/rules/lib/repo/utils#maybe) for doing something something similar, or would that only catch an exisiting llvm distribution with the exact same version?

Reply:
No your right maybe would serve the same purpose hear and is probably a little more idiomatic! Fixed